core/src/main/java/org/apache/gravitino/catalog/SchemaOperationDispatcher.java

```diff
@@ -336,6 +337,15 @@ public boolean dropSchema(NameIdentifier ident, boolean cascade) throws NonEmpty
             store.delete(ident, SCHEMA, cascade);
```

Comment:
How about changing this `cascade` to `true`? We should always delete the schema if the code reaches to here.

Reply:
I second you.

Reply:
that means we will maybe miss warning log:
```
LOG.warn("The schema to be dropped does not empty in the store: {}", ident, e);
```

Is this acceptable?

Reply:
It's fine. Besides, we can add the warning log in the storage layer, this can also notify the users.